src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests.cs

```diff
@@ -1437,7 +1513,10 @@ public async Task TestInTopLevelStatements()
                 """
                 using System.Collections.Generic;
 
-                var list = new List<int> { 1 };
+                var list = new List<int>
+                {
+                    1
+                };
```

Comment:
this was actually a bug from before that got fixed.  top level statemnets weren't properly wrapping (like what happens in a normal statement).